src/App.js

```diff
@@ -49,20 +46,22 @@ class App extends React.Component {
   };
 
   handleDelete = async (please) => {
-    console.log(please);
-
     const deleteURL = "http://localhost:3001/books";
     await axios.delete(deleteURL + "/" + please._id);
     this.getBooks();
   };
 
-  handleEdit = async () => {
+  hideEdit = () => {
     this.setState({
-      showEditForm: true
-    })
-    // const editURL = "http://localhost:3001/books";
-    // await axios.put(editURL + "/" + please._id);
-    // this.getBooks();
+      showEditForm: false,
+    });
+  };
+
+  showEdit = async (book) => {
+    this.setState({
+      showEditForm: true,
+      editBookid: book._id,
+    });
```

Comment:
Feeback here: I am guessing editBook is that you are updating the book...perhaps try renaming the method to `updateBook`. So far you have `getBooks` so maybe updateBook makes sense here.